phoenicis-apps/src/main/java/org/phoenicis/apps/BackgroundRepository.java

```diff
@@ -59,4 +59,19 @@ public void getScript(List<String> path, Consumer<ScriptDTO> callback, Consumer<
     public void setFilter(CombinedAppsFilter filter) {
         delegatedRepository.setFilter(filter);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
The equals and hashCode methods were autogenerated by intellij.
I'm not sure how to use EqualsBuilder and HashcodeBuilder do you have an example for them, or can you change them?